db/migrate/20210914114029_add_shadow_to_decidim_user.decidim.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+# This migration comes from decidim (originally 20210803131457)
+
+class AddShadowToDecidimUser < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
Hi, shouldn't we use `up` rather than `change` for adding a new column ? 
And so shouldn't we add a new method `down` for removing the newly created column in case of rollback ? 

```suggestion
  def up
```